docs/docs/nitrogen.md

```diff
@@ -119,22 +119,22 @@ Now run nitrogen:
 <Tabs groupId="package-manager">
   <TabItem value="npm" label="npm" default>
     ```sh
-    npm run nitro-codegen
+    npx nitrogen
```

Comment:
This was making assumptions on the existence of a "nitro-codegen" script in the package, which is not way I believe was the intent.